src/main/java/org/springframework/data/couchbase/core/mapping/CouchbaseSimpleTypes.java

```diff
@@ -28,16 +28,13 @@
 
 public abstract class CouchbaseSimpleTypes {
 
-  static {
-    Set<Class<?>> simpleTypes = new HashSet<Class<?>>();
-    simpleTypes.add(RawJsonDocument.class);
-    simpleTypes.add(JsonArray.class);
-    simpleTypes.add(Number.class);
-    COUCHBASE_SIMPLE_TYPES = Collections.unmodifiableSet(simpleTypes);
-  }
+  private static final Set<Class<?>> COUCHBASE_SIMPLE_TYPES = Stream.of(
```

Comment:
This changes the functionality, the simple types added before were 
static {
    Set<Class<?>> simpleTypes = new HashSet<Class<?>>();
    simpleTypes.add(CouchbaseDocument.class);
    simpleTypes.add(CouchbaseList.class);
    COUCHBASE_SIMPLE_TYPES = Collections.unmodifiableSet(simpleTypes);
  }